website/project/model.py

```diff
@@ -4290,15 +4292,30 @@ def _email_template_context(self, user, node, is_authorizer=False, urls=None):
 
     def _add_success_logs(self, node, user):
         src = node.registered_from
+        log_type = NodeLog.PROJECT_REGISTERED
+        submitted_time = None
+        try:
+            draft = DraftRegistration.find_one(Q("registered_node", "eq", node))
+            is_prereg = "Prereg" in draft.registration_schema.name
+            if is_prereg:
+                log_type = NodeLog.PREREG_CHALLENGE
+                submitted_time = iso8601format(draft.approval.initiation_date)
+        except NoResultsFound:
+            pass
+
+        params = {
+            'parent_node': src.parent_id,
+            'node': src._primary_key,
+            'registration': node._primary_key,
+        }
+        if submitted_time:
+            params.update({'submitted_time': submitted_time})
```

Comment:
You could move the `params` assignment up above the `try/except`, since it doesn't depend on `draft`. Then you could do the `params.update` in the `else` block (see https://github.com/CenterForOpenScience/osf.io/pull/5054#discussion_r55092124) and drop `submitted_time`.